tn_api_pricing_booking_spec.yml

```diff
@@ -862,14 +857,40 @@ components:
                             type: string
                             description: Unique Identifier of a Universal Record. 
                             example: "AXY67G"
-                        ticketed:       
-                            type: boolean
-                            description: Notes whether the booking has been ticketed. 
-                            example: true
+                        primary_passenger:
```

Comment:
Would it make sense to just have 'passenger_first_name' and 'passenger_last_name' at the root level? Would make sorting etc easier

Reply:
The reason I did it like this initially was to make it expandable to passenger details in the next API. But if it's gonna be a pain on QT I can for sure change that.

Reply:
I think the way it is makes sense if we are looking to add more parameters here in the future